### PR TITLE
mouseless 0.2.1 (new cask)

### PR DESCRIPTION
**Pull request by unitof**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by unitof** [comment]
I could use some help with the `livecheck`. This app seems to have a appcast endpoint somwhere at the domain `api.sonuscape.net`, but it verifies the SSL handshake so refuses to connect when it finds a self-signed certificate.

Screenshot 2024-12-15 at 8 36 31 PM

> HTTPSConnectionPool(host='[host]', port=443): Max retries exceeded with url: [latest] (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate in certificate chain (_ssl.c:1006)')))

Same error with trusted SSL proxies in in Charles and Proxyman.

Any ways around this, or alternative ways to inspect where it is trying to connect?

**Comment by unitof** [comment]
As a workaround, using regex on the homepage, which loads a JS module with matching version parameter.

However, would much prefer to get the Sparkle livecheck working if someone can help me figure out how to inspect a GET request from an app which verifies SSL handshakes.

**Comment by khipp** [comment]
It seems that upstream publishes releases in the [issue repository](https://github.com/croian/mouseless-issues/releases), which we can use in the `livecheck`.

There also is a documentation entry regarding SSL errors on startup, which might be helpful: https://github.com/croian/mouseless-issues/blob/main/mouseless_documentation.md#ssl--network-proxies.

File: Casks/m/mouseless.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+cask "mouseless" do
+  version "0.2.1"
+  sha256 "4007da3f7f002531f5e57ef419a1648872aa17729179b6f78aaac57a255934e1"
+
+  url "https://mouseless.click/mouseless-installer_v#{version}.dmg"
+  name "mouseless"
+  desc "Lightning-fast mouse control with the keyboard"
```
Comment (bevanjkay) [inline Casks/m/mouseless.rb:7]:
```suggestion
  desc "Mouse control with the keyboard"

**Review by khipp — APPROVED** [review]
Thank you, @unitof!

**Comment by croian** [comment]
thank you very much for making this, @unitof!  i was going through the process of doing it myself when I found the mouseless file 😂

please let me know if there's anything i can do for future maintenance.  or will brewTestBot handle it?  i'm not super familiar with the homebrew system

thanks again!

**Pull request merged**